src/nn/graph_builder.ts

```diff
@@ -75,13 +76,37 @@ export enum MLAutoPad {
  * [spec](https://webmachinelearning.github.io/webnn/#dictdef-mlconv2doptions)
  */
 export interface MLConv2dOptions {
+  padding?: [number, number, number, number];
+  strides?: [number, number];
+  dilations?: [number, number];
+  outputPadding?: [number, number];
+  autoPad?: MLAutoPad;
+  groups?: number;
+  inputLayout?: MLInputOperandLayout;
+  filterLayout?: MLFilterOperandLayout;
+  bias?: MLOperand;
+  activation?: MLOperator;
+}
+
+/**
+ * [spec](https://webmachinelearning.github.io/webnn/#enumdef-mlfilteroperandlayout)
+ */
+export enum MLConvTranspose2dFilterOperandLayout {
+  'iohw' = 'iohw',
+  'hwoi' = 'hwoi',
+  'ohwi' = 'ohwi',
+}
+
+/**
+ * [spec](https://webmachinelearning.github.io/webnn/#dictdef-mlconvtranspose2doptions)
+ */
+export interface MLConvTranspose2dOptions {
   padding?: [number, number, number, number];
   strides?: [number, number];
   dilations?: [number, number];
   outputPadding?: [number, number];
   outputSizes?: [number, number];
   autoPad?: MLAutoPad;
-  transpose?: boolean;
   groups?: number;
   inputLayout?: MLInputOperandLayout;
   filterLayout?: MLFilterOperandLayout;
```

Comment:
Please change `MLFilterOperandLayout` to `MLConvTranspose2dFilterOperandLayout`.